main.py

```diff
@@ -10,14 +10,50 @@
 NUSCENES_ROOT = 'nuscenes'
 
 SENSOR_NAMES = {
-    'LIDAR_TOP': 'lidar'
+    'LIDAR_TOP': 'lidar',
```

Comment:
do you mind adding a small inline comment above `SENSOR_NAMES` explaining the mapping (from nuscenes to argoverse, for example, nuscenes sensor on left, corresponding argoverse sensor on right)